packages/expo-cli/src/commands/credentials.ts

```diff
@@ -20,8 +25,23 @@ export default function (program: CommanderStatic) {
     .description('Manage your credentials')
     .helpGroup('credentials')
     .option('-p --platform <platform>', 'Platform: [android|ios]', /^(android|ios)$/i)
+    .option('--config [file]', 'Specify a path to app.json or app.config.js')
```

Comment:
we should move the `exp.ts` logic into a method and attempt to reuse it since this is a lot of stylistic logic. Perhaps we could add a way to skip project validation?

Reply:
Sure, skip flag would be probably best solution.

I decided to go this way because it's a bit isolated use cases and we will probably remove that command when web gui will be ready, so i didn't want modify code that handles other command. I don't have strong opinion on that either way.